geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreFactory.scala

```diff
@@ -133,20 +132,15 @@ object CassandraDataStoreFactory {
   val Description = "Apache Cassandra\u2122 distributed key/value store"
 
   object Params extends GeoMesaDataStoreParams {
+
+    override protected def looseBBoxDefault = false
+
     val ContactPointParam = new GeoMesaParam[String]("cassandra.contact.point", "HOST:PORT to Cassandra", optional = false, deprecatedKeys = Seq("geomesa.cassandra.contact.point"))
     val KeySpaceParam     = new GeoMesaParam[String]("cassandra.keyspace", "Cassandra Keyspace", optional = false, deprecatedKeys = Seq("geomesa.cassandra.keyspace"))
     val CatalogParam      = new GeoMesaParam[String]("cassandra.catalog", "Name of GeoMesa catalog table", optional = false, deprecatedKeys = Seq("geomesa.cassandra.catalog.table"))
     val UserNameParam     = new GeoMesaParam[String]("cassandra.username", "Username to connect with", deprecatedKeys = Seq("geomesa.cassandra.username"))
     val PasswordParam     = new GeoMesaParam[String]("cassandra.password", "Password to connect with", password = true, deprecatedKeys = Seq("geomesa.cassandra.password"))
 
-    override val LooseBBoxParam: GeoMesaParam[java.lang.Boolean] = {
```

Comment:
Just so that I know for later, can you elaborate/remind me about the C* story here?

Reply:
with our other data stores, we use the z-filter for our loose bbox queries, so they're still fairly accurate. with c* we don't do that, so they tend to be pretty inaccurate. so i defaulted c* to not use loose bboxes a little while ago - this change just makes it easier for other datastores to do the same thing without the ugly copying of the param